notebook/static/base/js/utils.js

```diff
@@ -879,4 +890,4 @@ define([
     };
 
     return utils;
-}); 
+}); 
```

Comment:
I removed the whitespace package...  This is probably remnant from a hand-revert of whitespace changes.